scanner/src/main/kotlin/scanners/scancode/ScanCodeResultParser.kt

```diff
@@ -81,27 +81,42 @@ internal fun parseResultsFile(resultsFile: File): JsonNode =
 
 /**
  * Generate a summary from the given raw ScanCode [result], using [startTime] and [endTime] metadata.
- * From the [scanPath] the package verification code is generated.
+ * From the [scanPath] the package verification code is generated. If [parseExpressions] is true, license findings are
+ * preferably parsed as license expressions.
  */
-internal fun generateSummary(startTime: Instant, endTime: Instant, scanPath: File, result: JsonNode) =
+internal fun generateSummary(
+    startTime: Instant,
+    endTime: Instant,
+    scanPath: File,
+    result: JsonNode,
+    parseExpressions: Boolean = true
```

Comment:
Does this need a default value? Is there a reason that the default here is `true` but in `ScanCode.kt` it's `false`?

Reply:
It's for convenience to avoid adapting all the calls in the tests. And as we agree that the tests should cover parsing the expressions, the default argument is `true` here; but on the "upper level" the default for the configuration is `false` for backward compatibility of our user's rules.